### DISCUSSION TITLE
Best way to distribute one-click-apps?

### DISCUSSION THREAD

**Discussion by adamshand** [caprover/caprover#1948] [General]
Hello,

I've just created a version of the n8n-io one click app which uses SQLite instead of Postgres.

I'm also about to create one for [linkding](https://github.com/sissbruecker/linkding).

Will PRs be accepted to the one-click-apps repo?  If not, what's the best way I can contribute these to the community?

Thanks!

PS.  Also just signed up for a year of Pro, thanks for all the great work!

**Comment by githubsaturn** [comment]
Hi Adam! 

For sure! Feel free to open a PR. My only 2 cents on n8n-io with SQLite is to make it clear why someone should pick one or the other.

**Reply by adamshand** [reply]
Okay, first contribution hopefully it all looks good! :-)

https://github.com/caprover/one-click-apps/pull/1031
